### ISSUE TITLE
Figure captions ...

### ISSUE THREAD

**Issue by chauff** [chauff/Web-Teaching#296]
Right now `<sup>` is used, there must be a better way ....

**Comment by maxwelld90** [comment]
HTML5 has a `<figcaption>` tag, maybe that is an option?
See [W3Schools](https://www.w3schools.com/tags/tag_figcaption.asp)

**Comment by chauff** [comment]
Yes! But then we would have to deviate from writing the lecture notes in Markdown and mix Markdown and HTML. I hope we could avoid that? Maybe there is a Jekyll plugin or something.

**Comment by maxwelld90** [comment]
Ah yes, of course, I forgot about that level. In that case, maybe we can find a plugin that changes the output HTML for a caption. Seems like the only logical way to do so. And completely agree about mixing HTML and Markdown. Defeats the purpose of MD in the first place.

**Comment by maxwelld90** [comment]
I found a quick and dirty way to do it… see this [StackOverflow](https://stackoverflow.com/a/19360305 <https://stackoverflow.com/a/19360305>) answer!

**Comment by maxwelld90** [comment]
But I think this is the answer you are looking for: [`jekyll-figure`](https://github.com/paulrobertlloyd/jekyll-figure <https://github.com/paulrobertlloyd/jekyll-figure>)

**Comment by chauff** [comment]
Thanks, I'll try it out!

**Comment by chauff** [comment]
Still ongoing ...
